examples/metrics/plot_gfrm.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skcosmo.datasets import load_degenerate_manifold
+from skcosmo.metrics import (
+    global_reconstruction_error,
+    global_reconstruction_distortion,
+)
+
+# Example for the usage of the :meth:`skcosmo.metrics.global_reconstruction_error` as GFRE and :meth:`skcosmo.metrics.global_reconstruction_distortion` as GFRD
```

Comment:
If you are going to write the examples as scripts, make sure they provide enough user output that the users can know what's going on. I'd prefer these as jupyter notebooks so that users can follow along directly, unless you plan on embedding them in the docpage.

Reply:
see c5ecd1c55bc28f568f0d9e6277b077897fe3e643